QUICKSTART.md

```diff
@@ -21,5 +21,5 @@ Reminders for the time when you already know what you're doing.
 
 ### Quick start a Nightking AMI build
 - Double-check the `experiments` folder to see if everything is there.
-- Run `packer build packer.json` with AWS access.
+- Run `NOAUTORUN=1 packer build packer.json` with AWS access.
```

Comment:
My usual workflow is to start an instance, not a build. Was wondering if the `NOAUTORUN` option is also available upon starting a Nightking instance scenario -- would be great if so! I guess this option can be passed, just as `DEBUG`, as a tag.

Reply:
DEV=1 will do that as a tag. NOAUTORUN is more for end-users with special needs. (Might be removed later.)